src/browser/netsurf.zig

```diff
@@ -2139,13 +2156,32 @@ pub inline fn documentCreateDocument(title: ?[]const u8) !*DocumentHTML {
     return doc_html;
 }
 
+pub inline fn documentCreateHTMLElement(doc: *Document, tag_name: []const u8) !*Element {
```

Comment:
Is there a reason to keep the originals? If our document is _always_ an html document, then it will _always_ create an HTMLElement. Only reason I can see to keep both is if we think we'll have a non-html document??